config/webpack.config.extract.js

```diff
@@ -23,7 +23,9 @@ module.exports = ({ filename }) => ({
             loader: 'postcss-loader',
             options: {
               config: {
-                path: path.join(__dirname, '../postcss.config.js')
```

Comment:
Postcss-load documentation says to not do that (it should be empty, or the path of a directory where the config file can be found if you really have some specific).